### PR TITLE
feat: add features

### PR DESCRIPTION
**Pull request by 0x74616e67**
1. update bg, update log
2. update popup dialog title
3. update i18n
4. remove 'English' switch
5. add open shortcut into dev env

---
This change is [Reviewable](https://reviewable.io/reviews/conflux-chain/helios/380)

### CODE REVIEW

File: packages/popup/src/pages/CurrentSeed/index.js [hunk 1]
```diff
@@ -66,12 +66,15 @@ function CurrentSeed() {
   const {setLoading} = useLoading()
 
   useEffect(() => {
-    setAccountNamePlaceholder(`Seed-1-${hdGroup[0]?.account?.length + 1}`)
+    setAccountNamePlaceholder(
+      `${t('seedPrefix')}1-${hdGroup[0]?.account?.length + 1}`,
```
Comment (angelia-yuqi-personal) [inline packages/popup/src/pages/CurrentSeed/index.js:70]: create account name with hardcode not i18n

File: packages/popup/src/pages/CreateSeed/NewSeed.js [hunk 2]
```diff
@@ -20,7 +20,8 @@ function NewSeed() {
   const [groupNamePlaceholder, setGroupNamePlaceholder] = useState('')
   const hdGroup = useHdAccountGroup()
   useEffect(() => {
-    setGroupNamePlaceholder(`Seed-${hdGroup.length + 1}`)
+    setGroupNamePlaceholder(`${t('seedPrefix')}${hdGroup.length + 1}`)
```
Comment (angelia-yuqi-personal) [inline packages/popup/src/pages/CreateSeed/NewSeed.js:23]: create account name with hardcode not i18n

File: packages/popup/src/pages/CurrentSeed/index.js [hunk 3]
```diff
@@ -66,12 +66,15 @@ function CurrentSeed() {
...
+    )
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [hdGroup])
   const onClickGroup = index => {
     setSelectedGroupIdx(index)
     setAccountNamePlaceholder(
-      `Seed-${index + 1}-${hdGroup[index].account.length + 1}`,
+      `${t('seedPrefix')}${index + 1}-${hdGroup[index].account.length + 1}`,
```
Comment (angelia-yuqi-personal) [inline packages/popup/src/pages/CurrentSeed/index.js:77]: create account name with hardcode not i18n

**Review by angelia-yuqi-personal — CHANGES_REQUESTED** [review]

File: packages/popup/src/pages/CurrentSeed/index.js (hunk 1 above)
Comment (0x74616e67) [reply packages/popup/src/pages/CurrentSeed/index.js]: done

File: packages/popup/src/pages/CreateSeed/NewSeed.js (hunk 2 above)
Comment (0x74616e67) [reply packages/popup/src/pages/CreateSeed/NewSeed.js]: done

File: packages/popup/src/pages/CurrentSeed/index.js (hunk 3 above)
Comment (0x74616e67) [reply packages/popup/src/pages/CurrentSeed/index.js]: done

**Review by angelia-yuqi-personal — APPROVED** [review]

**Pull request merged**